node/src/Commands.ts

```diff
@@ -1052,6 +1052,18 @@ export function createZDiffWithScores(keys: string[]): command_request.Command {
     return createCommand(RequestType.ZDiff, args);
 }
 
+/**
+ * @internal
+ */
+export function createZDiffStore(
+    destination: string,
+    keys: string[],
+): command_request.Command {
+    let args: string[] = [destination, keys.length.toString()];
+    args = args.concat(keys);
```

Comment:
optionally you can do
```suggestion
    const args: string[] = [destination, keys.length.toString(), ...keys];
```